.github/labeler-config.yml

```
@@ -5,6 +5,14 @@ config-change:
     - 'docs/config/*'
     - 'docs/service_specs/*'
     - 'docs/README.md'
+    - 'DotNet/**/appsettings.json'
+    - 'Java/**/application*.yml'
+
+# Add 'kafka-topic-change' label to any changes to the topics.txt file
+kafka-topic-change:
+- changed-files:
```

YAMLlint (1.37.1)

[error] wrong indentation: expected at least 1

(indentation)